### PR TITLE
chore: go code checker in travis

### PR DESCRIPTION
**Pull request by cd1989**
**What this PR does / why we need it**:

Add gofmt, golint, govet for go code in travis

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @zhujian7 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,23 +6,25 @@ services:
...
   - go build -o workflow-controller github.com/caicloud/cyclone/cmd/workflow/controller
   - go build -o workflow-coordinator github.com/caicloud/cyclone/cmd/workflow/coordinator
 
 script:
-  - go test -cover $(go list ./pkg/... ./cmd/... | grep -v '/pkg/k8s/' | grep -v '/pkg/apis/')
-  - travis_wait 15 overalls -project=github.com/caicloud/cyclone -covermode=count -debug -ignore=Godeps,vendor,tests,.git,scripts,node_modules,docs,tools,build,hack,manifests,release,web,pkg/k8s,pkg/apis
+  - ./tests/travis/gofmt.sh
+  - overalls -project=github.com/caicloud/cyclone -covermode=count -debug -ignore=Godeps,vendor,tests,.git,scripts,node_modules,docs,tools,build,hack,manifests,release,web,pkg/k8s,pkg/apis
+  - go vet  $(go list ./pkg/... ./cmd/... | grep -v "/pkg/k8s/")
+  - golint $(go list ./pkg/... ./cmd/... | grep -v "/pkg/k8s/")
```
Comment (supereagle) [inline .travis.yml]: [gometalinter](https://github.com/alecthomas/gometalinter)  is a more powerful linter tool.

File: pkg/server/handler/stage.go [hunk 2]
```diff
@@ -132,10 +132,10 @@ func receiveContainerLogStream(workflowrun, stage, container, namespace string,
 	}
 }
 
-// GetContainerLogStream gets real-time log of container within stage.
+// GetContainerLogStream ... GetContainerLogStream gets real-time log of container within stage.
```
Comment (zhujian7) [inline pkg/server/handler/stage.go]: Please delete `... GetContainerLogStream `

File: pkg/server/handler/stage.go [hunk 3]
```diff
@@ -208,19 +208,17 @@ func getContainerLogStream(workflowrun, stage, container, namespace string, ws *
 			ws.SetWriteDeadline(time.Now().Add(websocketutil.WriteWait))
 		}
 	}
-
-	return nil
 }
 
-// GetContainerLogs handles the request to get container logs, only supports finished stage records.
+// GetContainerLogs ... GetContainerLogs handles the request to get container logs, only supports finished stage records.
```
Comment (zhujian7) [inline pkg/server/handler/stage.go]: Same as above

**Pull request merged**